example-locale/src/config/index.js

```diff
@@ -0,0 +1,11 @@
+const Appwrite = require("appwrite"); //Import appwrite 
+
+let sdk  = new Appwrite();
+sdk
+  .setEndpoint("http://localhost:443/v1") // Your API Endpoint
+  .setProject("[PROJECT ID]") // Your project ID
+  .setKey("[SECRET API KEY]"); // Your secret API key
```

Comment:
We dont require a key for the client side sdk.

Reply:
Made the changes.

Reply:
The key field is still present ?

Reply:
```suggestion
```